src/scrapers/base-beinleumi-group.ts

```diff
@@ -298,11 +298,24 @@ async function getAccountIdsBySelector(page: Page): Promise<string[]> {
   return accountsIds;
 }
 
+async function getTransactionsFrame(page: Page): Promise<Frame> {
+  await sleep(5000);
+  const frames = page.frames();
+  const targetFrame = frames.find(f => f.name() === 'iframe-old-pages' || f.url().includes('some-part-of-url'));
+
+  if (!targetFrame) {
+    throw new Error('Target iframe not found');
+  }
```

Comment:
Maybe more detailed and less technical message?

`iframe name='iframe-old-pages' with URL includes 'some-part-of-url' was not found on the page to login with`

Reply:
Changed